mne/datasets/eegbci/eegbci.py

```diff
@@ -1,16 +1,24 @@
-# Author: Martin Billinger <martin.billinger@tugraz.at>
+# Authors: Martin Billinger <martin.billinger@tugraz.at>
+#          Daniel McCloy <dan@mccloy.info>
 # License: BSD Style.
 
 import os
 from os import path as op
+import re
+import pkg_resources
 
 from ..utils import _get_path, _do_path_update
-from ...utils import _fetch_file, _url_to_local_path, verbose
+from ...utils import _fetch_file, _url_to_local_path, verbose, deprecated
+from ...utils.check import _soft_import
+
+pooch = _soft_import('pooch', 'dataset downloading', True)
 
 
 EEGMI_URL = 'https://physionet.org/files/eegmmidb/1.0.0/'
 
 
+@deprecated('mne.datasets.eegbci.data_path() is deprecated and will be removed'
```

Comment:
@drammock should I deprecate these functions for these datasets in #9742 ?

Reply:
I can't recall why I was deprecating here (maybe to improve API consistency?)

Reply:
maybe for the first PR you should try to minimize disruption (try to swap in pooch behind the scenes, and change the user-facing API minimally if at all)